services/cinema_hall.py

```diff
@@ -8,8 +8,12 @@ def get_cinema_halls() -> QuerySet:
 
 
 def create_cinema_hall(
-    hall_name: str, hall_rows: int, hall_seats_in_row: int
+        hall_name: str,
+        hall_rows: int,
+        hall_seats_in_row: int
 ) -> CinemaHall:
     return CinemaHall.objects.create(
```

Comment:
You don’t need to `return` anything in create/update/delete methods, check everywhere

Reply:
okay, but this code was written not by me)
And in the task description it is nowhere said that I need to change the existing code.
Maybe, it is better to mention it in the task.